NICE.Identity.Management/ClientApp/src/components/App/App.tsx

```diff
@@ -1,25 +1,46 @@
 import React from "react";
+
+import { BrowserRouter as Router, Route, Link } from "react-router-dom";
+import { Header, Footer } from "@nice-digital/global-nav";
+import { Breadcrumbs, Breadcrumb } from "@nice-digital/nds-breadcrumbs";
+
 import { UsersList } from "../../views/UsersList/UsersList";
 import { User } from "./../../views/User/User";
 import { UserRoles } from "./../../views/UserRoles/UserRoles";
-import { BrowserRouter as Router, Route, Link } from "react-router-dom";
 
-class App extends React.Component {
+export class App extends React.Component {
 	render() {
 		return (
 			<Router>
-				<nav>
-					<ul>
-						<li>
-							<Link to="/">Home</Link>
-						</li>
-						<li>
-							<Link to="/users">Users</Link>
-						</li>
-					</ul>
-				</nav>
+				<Header search={false} />
```

Comment:
We should override the skipLinkTarget prop to point to our main heading that sits below the breadcrumb so a screenreader user wouldn't have to skip over breadcrumbs on every page.